### PR TITLE
#9: Support deplyoment to Maven Central

### PR DESCRIPTION
**Pull request by redcatbear**
Enhancements:
* #9: Support deployment to Maven Central

Bugfix:
* #9: Fixed JavaDoc

### CODE REVIEW

**Comment by redcatbear** [comment]
@kaklakariada, please review.

File: .classpath [hunk 1]
```diff
@@ -1,17 +1,46 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (kaklakariada) [inline .classpath]: Do you want to check in .classpath and .project? This is generated by eclipse when importing a maven project.

**Review by kaklakariada — APPROVED** [review]

**Review by kaklakariada — CHANGES_REQUESTED** [review]
Document how to configure bintray credentials and how to publish (just copy the instructions from oft).

File: .classpath (hunk 1 above)
Comment (redcatbear) [reply .classpath]: Removed and added to `.gitignore`.

**Comment by redcatbear** [comment]
Added deployment info in `README.md`.

**Comment by redcatbear** [comment]
@kaklakariada, please review again.

File: README.md [hunk 2]
```diff
@@ -95,10 +95,39 @@ The list below show all build time dependencies in alphabetical order. Note that
...
+1. Run command
+
+    ```bash
+    mvn deploy
+    ```
+1. Merge to `master` branch
+1. Create a [release](https://github.com/itsallcode/junit5-system-extensions/releases) of the `master` branch on GitHub.
+1. Sign in at [bintray.com](https://bintray.com)
+1. Go to the [Bintray project page](https://bintray.com/itsallcode/itsallcode/junit5-system-extensions)
+1. There should be a notice saying "You have 6 unpublished item(s) for this package". Click the "Publish" link. Binaries will be available for download at [JCenter](https://jcenter.bintray.com/org/itsallcode/junit5-system-extensions/)
```
Comment (kaklakariada) [inline README.md]:
```suggestion
```

We have `publish=1` in the repo url in `pom.xml`, so artifacts are published automatically.

**Review by kaklakariada — APPROVED** [review]

**Pull request merged**